markets/perps-market/contracts/modules/MarketConfigurationModule.sol

```diff
@@ -24,6 +24,18 @@ contract MarketConfigurationModule is IMarketConfigurationModule {
         config.settlementStrategies.push(strategy);
     }
 
+    function setSettlementStrategyEnabled(
+        uint128 marketId,
+        uint256 strategyId,
+        bool isEnabled
+    ) external override {
+        PerpsMarket.load(marketId).onlyMarketOwner();
+        PerpsMarketConfiguration
+            .load(marketId)
+            .settlementStrategies[strategyId]
+            .disabled = isEnabled;
```

Comment:
this isn't right.  should be `!isEnabled`

the functionality is exactly the same as here: https://github.com/Synthetixio/synthetix-v3/blob/e841287cf55ca5b687ce6e8660d31da81b4253d4/markets/spot-market/contracts/modules/AsyncOrderConfigurationModule.sol